packages/e2e-tests/src/config/bootstrap.js

```diff
@@ -118,7 +118,7 @@ if ('true' === process.env.CI) {
 }
 
 // Set default timeout for individual expect-puppeteer assertions. (Default: 500)
```

Comment:
```suggestion
// Set default timeout for individual expect-puppeteer assertions. (Default: 1000)
```

Reply:
The default in `expect-puppeteer` is `500`, so this is a reference to that.